src/Components/OATPropertyEditor/Internal/PropertiesModelSummary.tsx

```diff
@@ -0,0 +1,380 @@
+        setModelPath(value);
+    }, []);
+    const onChangeRelationshipName = useCallback((_ev, value: string) => {
+        // generally banned characters
+        INVALID_CHARACTERS.forEach((x) => (value = value.replaceAll(x, '')));
+        // specially banned for names
+        [':'].forEach((x) => (value = value.replaceAll(x, '')));
+        setRelationshipName(value);
+    }, []);
+
+    // needed primarly for the version spinner since it behaves differently and you don't have to set focus
+    const forceUpdateId = useCallback(
+        ({ namespace, modelName, path, version }: IPartialModelId) => {
+            const newId = buildModelId({
+                namespace: namespace || parsedId.namespace,
+                modelName: modelName || modelUniqueName,
+                path: path || modelPath,
+                version: Number(version || modelVersion)
+            });
+            commitModelIdChange(newId);
+        },
+        [
+            commitModelIdChange,
+            modelPath,
+            modelUniqueName,
+            modelVersion,
+            parsedId.namespace
+        ]
+    );
+
+    // side effects
+    // when selected item changes, update all the states
+    useEffect(() => {
+        const parsedId = parseModelId(selectedItem?.['@id']);
+        setModelUniqueName(parsedId.name);
+        setModelPath(parsedId.path);
+        setModelVersion(parsedId.version);
+        setRelationshipName(
+            isDTDLReference(selectedItem) ? selectedItem?.name : ''
+        );
+    }, [selectedItem]);
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useExtendedTheme()
+    });
+
+    logDebugConsole('debug', 'Render {item}', selectedItem);
+    if (!selectedItem) {
+        console.warn(
+            '[PROPERTIES_MODEL_SUMMARY] No selected item provided, aborting render'
+        );
+        return null;
+    }
+    return (
+        <>
+            <Stack
+                styles={classNames.subComponentStyles.rootStack}
+                tokens={{ childrenGap: 8 }}
+            >
+                {/* HEADER */}
+                <ModelPropertyHeader
+                    entityId={itemId}
+                    entityName={
+                        isModelSelected
+                            ? modelUniqueName
+                            : isReferenceSelected
+                            ? relationshipName
+                            : ''
+                    }
+                    entityType={
+                        selectedItem ? selectedItem['@type'].toString() : ''
+                    }
+                    onInfoButtonClick={setIsInfoModalOpenTrue}
+                />
+
+                {isReferenceSelected && (
+                    <>
+                        {/* NAME SECTION */}
+                        <div className={classNames.row}>
+                            <Text
+                                id={'oat-relationship-name'}
```

Comment:
string doesn't need `{}`

Reply:
I think our current style rule is to always use {} on props. Is there a benefit or difference to not having them?

Reply:
Early guidance was that `{}` gets treated as JS code and therefore has a "cost". It looks like that's not so much the case anymore so this is a non-issue.